src/common/MovieList/index.js

```diff
@@ -2,7 +2,13 @@ import {MovieTile} from "./MovieTile";
 
 function MovieList() {
     return (
-        <MovieTile />
+        <MovieTile
+            title={"Mulan"}
+            year={"2020"}
+            genre={"Action"} //na podstawie tego w przyszłości będzie renderować te boksy z gatunkami
```

Comment:
**Piszę to tylko jako notatkę!
W przyszłości można tu przekazywać właśnie obiekt/tablicę z gatunkami które chcemy wyrenderować.
czyli np genre={["Action", "Drama", "Cośtam"]}